src/lib/libcmd/iconv.c

```diff
@@ -68,7 +68,7 @@ USAGE_LICENSE
 ;
 
 #include <cmd.h>
-#include <iconv.h>
+#include <ast_iconv.h>
```

Comment:
We are not building this file yet.

Reply:
Huh? You removed it from `iffe` generation and made it a static header back in commit 6071c35c6d. And it's already being used in several other modules such as *src/lib/libast/tm/tmlocale.c*.

Reply:
I was referring to this file `src/lib/libcmd/iconv.c`

Reply:
Oh, right, I noticed that as well. I'm simply changing this include for consistency so we don't have a "WTF?" moment if and when we do build this. And if we ultimately drop this code I don't feel bad about throwing away a mechanical one line change :smile: